web/profiles/joinup/joinup.info.yml

```diff
@@ -57,17 +58,21 @@ config_devel:
   - block.block.seven_page_title
   - block.block.seven_primary_local_tasks
   - block.block.seven_secondary_local_tasks
+  - block.block.userlogin
   - core.date_format.fallback
-  - core.date_format.html_datetime
   - core.date_format.html_date
+  - core.date_format.html_datetime
   - core.date_format.html_month
   - core.date_format.html_time
   - core.date_format.html_week
-  - core.date_format.html_yearless_date
   - core.date_format.html_year
+  - core.date_format.html_yearless_date
   - core.date_format.long
   - core.date_format.medium
   - core.date_format.short
+  - core.entity_form_display.rdf_entity.collection.propose
```

Comment:
Can we move this to collection module install config files? Isn't it better to keep it with the rest of the collection config files?